tasks/request_files/request_files.py

```diff
@@ -138,19 +139,32 @@ def task(
         )
         retry_sleep_secs = DEFAULT_RESTORE_RETRY_SLEEP_SECS
 
-    # Get retrieval type
+    # Get retrieval type value from config first
     try:
-        retrieval_type = os.environ[OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY]
-        if retrieval_type not in ("Standard", "Bulk", "Expedited"):
-            msg = (
-                f"Invalid RESTORE_RETRIEVAL_TYPE: '{retrieval_type}'"
-                " defaulting to 'Standard'"
-            )
-            LOGGER.info(msg)
-            retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
+        retrieval_type = event["config"][CONFIG_RESTORE_RETRIEVAL_TYPE_KEY]
+        LOGGER.info(f"Found the following retrieval type from config: {retrieval_type}")
     except KeyError:
-        LOGGER.warn("Invalid RESTORE_RETRIEVAL_TYPE: 'None' defaulting to 'Standard'")
-        retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
+        LOGGER.warning(
+            f"{CONFIG_RESTORE_RETRIEVAL_TYPE_KEY} is not set. Using RESTORE_RETRIEVAL_TYPE environment value."
+        )
+        retrieval_type = None
+
+    if retrieval_type is None or retrieval_type not in ("Standard", "Bulk", "Expedited"):
+        error_msg = f"Invalid value in {CONFIG_RESTORE_RETRIEVAL_TYPE_KEY}: '{retrieval_type}'. Getting the value from env variable"
+        LOGGER.warning(error_msg)
+        # Get retrieval type from env variable
+        try:
+            retrieval_type = os.environ[OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY]
+            if retrieval_type not in ("Standard", "Bulk", "Expedited"):
```

Comment:
This does not need to be in the `try` block. Put everything from `if` to `retrieval_type = ...` outside the `try/except`, and replace the set in the except with `retrieval_type = None`

Reply:
I concur with @AndrewDorn see previous comment on simplifying.

Reply:
changed to 

```
    if retrieval_type is None or retrieval_type not in ("Standard", "Bulk", "Expedited"):
        LOGGER.warn(
            f"Invalid value in {CONFIG_RESTORE_DEFAULT_RETRIEVAL_TYPE_OVERRIDE_KEY}: '{retrieval_type}'. Getting the value from env variable"
        )
        # Get retrieval type from env variable
        try:
            retrieval_type = os.environ[OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY]
        except KeyError:
            LOGGER.warn("Invalid {OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY} value defaulting to {DEFAULT_RESTORE_RETRIEVAL_TYPE}")
            retrieval_type = None
        if retrieval_type not in ("Standard", "Bulk", "Expedited"):
            LOGGER.warn("Invalid {OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY} value defaulting to {DEFAULT_RESTORE_RETRIEVAL_TYPE}")
            retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
```